### ISSUE TITLE
Cannot add tags to post

### ISSUE THREAD

**Issue by frantzypants** [TryGhost/Ghost#9619]
### Issue Summary

When creating a new post, or editing a draft/published post, I am unable to create **new tags** from the post settings menu, however adding existing tags works fine. 

![console](https://puu.sh/AixyO/7c0a8c9905.png)

### Steps to Reproduce

1. Create new post / Edit existing post
2. Expand the post settings menu
3. Begin typing a new tag and hit enter
4. Nothing happens, the tag is not assigned/created

Expected behaviour would be that hitting enter would create the new tag and assign it to the post as it had once worked before.

### Technical details:

* Ghost Version: v1.22.5
* Browser/OS: Chrome (latest) - Mac OS v10.13.3

**Comment by healinginfluence** [comment]
I am having the same issue.

**Comment by mrfinks** [comment]
Had same issue, found that forcing `_findTagByName` to return a promise would get past this.

[https://github.com/TryGhost/Ghost-Admin/blob/cd563b66b7fd825968de4149d9f06c1e1d44e20f/app/components/gh-psm-tags-input.js#L90](https://github.com/TryGhost/Ghost-Admin/blob/cd563b66b7fd825968de4149d9f06c1e1d44e20f/app/components/gh-psm-tags-input.js#L90)

changed:
```
    _findTagByName(name) {
        let withMatchingName = function (tag) {
            return tag.get('name').toLowerCase() === name.toLowerCase();
        };
        return this.get('availableTags').then(availableTags => availableTags.find(withMatchingName));
    }
```

to:
```
    _findTagByName(name) {
        let withMatchingName = function (tag) {
            return tag.get('name').toLowerCase() === name.toLowerCase();
        };
        return Promise.resolve(this.get('availableTags')).then(availableTags => availableTags.find(withMatchingName));
    }
```
wrapping in a try/catch worked too.

Disclaimer: I am definitely an ember noob, so apologies if this code scares you.

**Comment by kevinansfield** [comment]
The fix for this has been released in [Ghost 1.22.7](https://github.com/TryGhost/Ghost/releases/tag/1.22.7)
